velox/connectors/tpch/TpchConnector.h

```diff
@@ -98,14 +98,20 @@ class TpchDataSource : public DataSource {
 
   velox::tpch::Table tpchTable_;
   size_t scaleFactor_{1};
+  size_t tpchTableRowCount_{0};
   RowTypePtr outputType_;
 
   // Mapping between output columns and their indices (channelIndex) in the
   // dbgen generated datasets.
   std::vector<ChannelIndex> outputColumnMappings_;
 
   std::shared_ptr<TpchConnectorSplit> currentSplit_;
+
+  // First (splitOffset_) and last (splitLimit_) row number that should be
+  // generated by this split.
   size_t splitOffset_{0};
+  size_t splitLimit_{0};
```

Comment:
nit:
Can we rename this to `splitEnd_` or `splitEndOffset_`? I often see C++ Std lib uses `begin`, `end` keywords but not `limit`. 
btw. Sorry for bring up many nits!